### PR TITLE
APM-5127: Increase connect, read, write timeout

### PR DESCRIPTION
**Pull request by coreyjackwilson**
### If applied, this pull request will...
- [x] increase connect, read, write timeout from 10s to 1m
 
### Addresses:
* [APM-5070](https://appneta.atlassian.net/browse/APM-5070)

### Questions
- 

### Links to any relevant articles or other resources-
* []()

@appneta/cat-astrophe

### CODE REVIEW

File: rundeck-server/src/main/resources/buildServerResources/rundeck-runner-config.jsp [hunk 1]
```diff
@@ -99,8 +99,15 @@
...
-
+    <tr>
+        <th>
+            <label for="runDeckWaitForJobTimeout">Wait for job to finish timeout:</label>
+        </th>
+        <td>
+            <div class="completionIconWrapper">
+                <props:textProperty name="runDeckWaitForJobTimeout" className="longField"/>
+            </div>
+            <span class="smallNote">How long to wait for the job to finish (if waiting is selected). Please note if waiting is no selected, this field is meaningless</span>
```
Comment (MagnusHvidsten) [inline rundeck-server/src/main/resources/buildServerResources/rundeck-runner-config.jsp]: NIT spelling; ~no~ not*

File: rundeck-agent/src/main/kotlin/com/hadihariri/teamcity/plugins/rundeck/agent/RunDeck.kt [hunk 2]
```diff
@@ -63,7 +64,11 @@ public object RunDeck {
         if (execution.code == 200) {
             println(ServiceMessage.asString("rundeck", mapOf("text" to "Job ${rundeckOptions.jobId} launched successfully with id ${execution.result}", "status" to "NORMAL")))
             if (rundeckOptions.waitFinish) {
-                while (counter < 1200) {
+                var timeout: Long = 2700
```
Comment (jeff-leong) [inline rundeck-agent/src/main/kotlin/com/hadihariri/teamcity/plugins/rundeck/agent/RunDeck.kt]: What timeout feature in rundeck does this control?

Comment (coreyjackwilson) [reply rundeck-agent/src/main/kotlin/com/hadihariri/teamcity/plugins/rundeck/agent/RunDeck.kt]: Its the timeout for teamcity waiting on the rundeck job to finish

**Comment by jeff-leong** [comment]
How is this going to be tested?

**Pull request merged**